### ISSUE TITLE
Crash after setting elevator in newbie to !RADIO

### ISSUE THREAD

**Issue by jalles76** [luciensadi/AwakeMUD#85]
Game crashed after 'push 4'

**Comment by jalles76** [comment]
Aaaand the elevator room now has an exit to room 0...

**Comment by jalles76** [comment]
after calling the elevator again it disappeard

**Comment by jalles76** [comment]
````
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x00000000005b47ae in close_elevator_doors (room=0x7fae1d3e8ce0, num=0, floor=0) at transport.cpp:705
705       if (world[rnum].dir_option[dir]->keyword)
(gdb) bt
#0  0x00000000005b47ae in close_elevator_doors (room=0x7fae1d3e8ce0, num=0, floor=0) at transport.cpp:705
#1  0x00000000005b5cf1 in process_elevator (room=0x7fae1d3e8ce0, ch=0x1475770, cmd=359, argument=0x7fff1b3650e5 "4") at transport.cpp:936
#2  0x00000000005b4eb7 in elevator_spec (ch=0x1475770, me=0x7fae1d3e8ce0, cmd=359, argument=0x7fff1b3650e4 " 4") at transport.cpp:803
#3  0x000000000051833d in special (ch=0x1475770, cmd=359, arg=0x7fff1b3650e4 " 4") at interpreter.cpp:1807
#4  0x0000000000516bd6 in command_interpreter (ch=0x1475770, argument=0x7fff1b3650e0 "push 4", tcname=0x14765f0 "Finster") at interpreter.cpp:1382
#5  0x00000000004ab1db in game_loop (mother_desc=3) at comm.cpp:679
#6  0x00000000004aa3c3 in init_game (port=4000) at comm.cpp:362
#7  0x00000000004a9ed4 in main (argc=3, argv=0x7fff1b365a38) at comm.cpp:238
````

**Comment by luciensadi** [comment]
This crash occurs when you have a 1-way exit from the elevator to the world (i.e. you hit 'redit' while the doors were open, then they closed, then you saved). The code traverses through the exit and attempts to access the exit in the other direction to close it, but since this exit doesn't exist it's a bad access that causes a crash.

**Comment by luciensadi** [comment]
Issue fixed in PR #88.
